LISAv2-Framework.psm1

```diff
@@ -137,12 +133,13 @@ function Start-LISAv2 {
 			}
 
 			# Validate test platform, and select test controller of the platform
-			$supportedPlatforms = @("Azure", "HyperV", "WSL")
 			if ($paramTable.ContainsKey("TestPlatform")) {
 				$testPlatform = $paramTable["TestPlatform"]
 			}
 			if ($testPlatform) {
-				if ($supportedPlatforms.contains($testPlatform)) {
+				$moduleName = "TestControllers\$($testPlatform)Controller.psm1"
```

Comment:
It looks ok, but can you also update document, https://github.com/LIS/LISAv2/blob/master/Documents/How-to-use.md? It seems each TestPlatform param's value should match to a part of plugin modules. Please add 'WSL', and inform out this should be matching in the document.

Reply:
doc update requires.

Reply:
Sure @juhlee-microsoft